.gitignore

```diff
@@ -74,3 +74,8 @@ ENV/
 
 # Fabric Files
 fabfile/
+
+# Frontend Modules
+node_modules
+frontend/bower_components
+frontend/node_modules
```

Comment:
@deshraj : we don't need these directories in main gitignore. We already have a different gitignore for frontend work.
@itaditya : please don't include any frontend related files/directories in main gitignore. Frontend specific gitignore is already present in frontend directory.